src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php

```diff
@@ -93,99 +93,26 @@ public function onPullAfterUpdate(Event $event)
 	 *
 	 * @since   1.0
 	 */
-	protected function addRTClabel($hookData, Github $github, Logger $logger, $project, IssuesTable $table)
+	protected function checkRTClabel($hookData, Github $github, Logger $logger, $project, IssuesTable $table)
 	{
-		// Validation, if the status isn't RTC then go no further
-		if ($table->status != 4)
-		{
-			return;
-		}
-
 		// Set some data
-		$RTClabel    = 'RTC';
-		$addLabels   = array();
-		$rtcLabelSet = false;
+		$label      = 'RTC';
+		$labels     = array();
+		$labelIsSet = $this->checkLabel($hookData, Github $github, Logger $logger, $project, IssuesTable $table, $label);
```

Comment:
You should remove type hinting here. And also do not forget to remove $table as it is not used.